### PR TITLE
Move awoken skill aliases to doc

### PR DESCRIPTION
**Pull request by chasehult**

### CODE REVIEW

File: crud/tables/awoken_skills.py [hunk 1]
```diff
@@ -138,8 +143,12 @@ async def edit(self, ctx, awoken_skill, *elements):
         [p]crud awokenskill edit misc_comboboost key1 "Value1" key2 "Value2"
         """
         dbcog = await self.get_dbcog()
-        if awoken_skill in dbcog.KNOWN_AWOKEN_SKILL_TOKENS:
-            awoken_skill_id = dbcog.KNOWN_AWOKEN_SKILL_TOKENS[awoken_skill].value
+        index = await dbcog.get_index(Server.COMBINED)
+        awo_alias_dict = {alias: awo_id
```
Comment (RheingoldRiver) [inline crud/tables/awoken_skills.py]: pull this out maybe? if you're doing it twice

File: dbcog/monster_index.py [hunk 2]
```diff
@@ -29,6 +29,7 @@
 TREE_MODIFIER_OVERRIDE_SHEET = SHEETS_PATTERN.format(1372419168)
 CONTENT_TOKEN_ALIAS_SHEET = SHEETS_PATTERN.format(1229125459)
 SERIES_OVERRIDES_SHEET = SHEETS_PATTERN.format(959933643)
+AWOKEN_SKILL_SHEET = SHEETS_PATTERN.format(1063132392)
```
Comment (RheingoldRiver) [inline dbcog/monster_index.py:32]: i am really sorry to say this but i think these 4 things should be in tsutils (they will never be changed, ever)

Comment (RheingoldRiver) [reply dbcog/monster_index.py:32]: but actually can we put these in an existing sheet? maybe the same one as dungeons? we have too many sheets

File: dbcog/monster_index.py [hunk 3]
```diff
@@ -175,14 +152,23 @@ async def setup(self, graph: MonsterGraph):
...
+        for data in series_data:
             if data['series_id'].isdigit():
                 sid = int(data['series_id'])
                 name = data['alias'].strip().lower()
                 if " " in name:
                     self.multi_word_tokens.add(tuple(name.lower().split(" ")))
                 self.series_id_to_pantheon_nickname[sid].add(name.lower().replace(" ", ""))
 
+        for data in awo_data:
+            if data['awoken_id'].isdigit():
```
Comment (RheingoldRiver) [inline dbcog/monster_index.py]: you can if not isdigit continue and clean up indentation a bit i think

**Review by RheingoldRiver — COMMENTED** [review]
made a couple small comments, really i just don't want ANOTHER separate gdoc, we have too many lol

all the id2 looks good yay!!!!!!!!!!!!!!!!!!!!

File: dbcog/monster_index.py (hunk 2 above)
Comment (chasehult) [reply dbcog/monster_index.py:32]: Will do in different PR, but it is in same sheet as everything but dungeons

**Pull request merged**